pay-api/src/pay_api/models/statement.py

```diff
@@ -115,14 +116,49 @@ def payment_methods(self):
 
         return list(payment_methods)
 
+    @hybrid_property
+    def amount_owing(self):
+        """Return amount owing on statement."""
+        from .statement_invoices import StatementInvoices  # pylint: disable=import-outside-toplevel
```

Comment:
not good

Reply:
removed hybrid property and updated search query